src/screens/staking-delegate-preview.tsx

```diff
@@ -42,7 +44,7 @@ export const StakingDelegatePreviewScreen = () => {
   }, [cosmos]);
 
   const onDone = useCallback(async () => {
-    if (wallet && wallet.isValid()) {
```

Comment:
I think we need to have `isValid`...

Reply:
isValid is realm only feature. We don't need it here
[Realm what is RealmObject.isValid?](https://stackoverflow.com/questions/42404508/realm-what-is-realmobject-isvalid)